packages/kbn-docs-utils/src/api_docs/build_api_docs_cli.ts

```diff
@@ -136,6 +139,24 @@ export function runBuildApiDocsCli() {
             group: 'References to deprecated APIs',
             value: pluginStats.deprecatedAPIsReferencedCount,
           },
+          {
+            id,
+            meta: { pluginTeam },
+            group: 'Eslint disable line counts',
```

Comment:
```suggestion
            group: 'ESLint disabled line counts',
```

Reply:
Fixed the casing here and the following instances. I also changed to `disabled` from `disable` as it past-tense.

Reply:
Thanks, these suggestions look much better.